docs/wagtail-pages.md

```diff
@@ -0,0 +1,207 @@
+from v1.models import CFGOVPage
+
+
+class LandingPage(CFGOVPage):
+    header = StreamField([
+        ('hero', molecules.Hero()),
+        ('text_introduction', molecules.TextIntroduction()),
+    ], blank=True)
+
+    # …
+
+    content_panels = CFGOVPage.content_panels + [
+        StreamFieldPanel('header'),
+        # …
+    ]
+```
+
+## Parent / child page relationships
+
+Wagtail provides two attributes to page models that enable restricting the types of subpages or parent pages a particular page model can have. 
+
+For example, in our [interactive regulations page models](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/regulations3k/models/pages.py#L138) 
+we have a `RegulationLandingPage` that can be created anywhere in the page tree. 
+`RegulationLandingPage`, however, can only have two types of pages created within it: 
+`RegulationPage` and `RegulationSearchPage`. 
+This parent/child relationship is expressed by setting `subpage_types` on `RegulationLandingPage`
+and `parent_page_types` on `RegulationPage` and `RegulationSearchPage` 
+to either the model classes themselves or to a model name in the form `app_label.ModelName`:
+
+```python
+from v1.models import CFGOVPage
+
+
+class RegulationLandingPage(CFGOVPage):
+    subpage_types = ['regulations3k.RegulationPage', 'RegulationsSearchPage']
+
+
+class RegulationsSearchPage(CFGOVPage):
+    parent_page_types = ['regulations3k.RegulationLandingPage']
+    subpage_types = []
+
+
+class RegulationPage(CFGOVPage):
+    parent_page_types = ['regulations3k.RegulationLandingPage']
+    subpage_types = []
+```
+
+!!! note 
+    Note: by setting `subpage_types` to an empty list on `RegulationPage` and `RegulationSearchPage`, 
+    we ensure that neither of these pages can have child pages.
+
+## Template rendering
+
+Sometimes new Wagtail page types will need to make customizations to their base template when rendering the page.
+This is done by overriding the `template` attribute on the page model.
+
+For example, the [interactive regulations landing page](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/regulations3k/models/pages.py#L138) 
+includes a customized list of recently issued notices that gets loaded dynamically from the Federal Register. 
+To do this it provides its own template that inherits from our base templates 
+and overrides the `content_sidebar` block to include a seperate `recent_notices` template:
+
+```python
+from v1.models import CFGOVPage
+
+
+class RegulationLandingPage(CFGOVPage):
+    template = 'regulations3k/landing-page.html'
+```
+
+And [`regulations3k/landing-page.html`](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/regulations3k/jinja2/regulations3k/landing-page.html#L5):
+
+```jinja2
+{% extends 'layout-2-1-bleedbar.html' %}
+
+{% import 'recent-notices.html' as recent_notices with context %}
+
+{% block content_sidebar scoped -%}
+    {{ recent_notices }}
+{%- endblock %}
+
+{% block javascript scoped %}
```

Comment:
This block is a little overwhelming and doesn't add much to the example. How would you feel about excluding it? The `content_sidebar` override demonstrates what is being described above.

Reply:
Excluded in 2b0f4a2.